integr8ly/cloud-resource-operator
# MGDAPI-5458 - refactor: remove blocking gcp calls

## Overview

Jira: issues.redhat.com/browse/MGDAPI-5458

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make run` against a GCP cluster
- Seed Postgres
```
PROVIDER=gcp make cluster/seed/postgres
```
* Verify phase is in progress during ip address range or connection service creation
```
oc get postgres example-postgres -n cloud-resource-operator -o json | jq '.status'
```
![image](https://user-images.githubusercontent.com/24636860/231484475-143d8b26-06f3-4f40-97af-3e84f42f1ab0.png)
* Delete postgres
- Seed Redis
```
PROVIDER=gcp make cluster/seed/redis
```
- Verify phase is in progress during ip address range or connection service creation
```
oc get redis example-redis -n cloud-resource-operator -o json | jq '.status'
```

![image](https://user-images.githubusercontent.com/24636860/231488360-b3e7bc18-733b-4e50-91a3-e0af9aebd1ce.png)
* Delete redis
## Checklist
